aspnetcore/fundamentals/configuration/index/samples/1.x/ConfigurationSample/Controllers/HomeController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using ConfigurationSample.Models;
+using ConfigurationSample.ViewModels;
+
+namespace ConfigurationSample.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly IConfiguration _config;
+
+        public HomeController(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public IActionResult Index()
+        {
+            var viewModel = new Config();
+
+            // Take a subset of the configuration entries because the 
+            // AddEnvironmentVariables call provided by CreateDefaultBuilder 
+            // doesn't have a prefix filter. All environment variables available 
+            // are provided to the app's configuration. Without the filtering
+            // applied here, the list of configuration entries shown by the app
+            // can number over 50.
+            var configEntryFilter = new string[] { "ASPNETCORE_", "urls", "Logging", "ENVIRONMENT", "contentRoot", "AllowedHosts", "applicationName", "CommandLine" };
+
+            var config = _config.AsEnumerable();
+
+            viewModel.FilteredConfiguration = config.Where(
+                kvp => config.Any(
+                    i => configEntryFilter.Any(prefix => kvp.Key.StartsWith(prefix))));
+
+            // Uncomment the next line to render all configuration key-value pairs.
```

Comment:
I recommend making this a TODO comment. Doing so surfaces the info in the Task List window of VS.

Reply:
This isn't really a "TODO" ... it's just telling the reader that if they want all of the env vars loaded that they just need to uncomment the line. I could say in-text in the topic how to do it (perhaps in a NOTE) and eliminate this from the sample app. Do you have a preference on how to handle it?

Reply:
I like your NOTE idea. I suspect most devs ignore the comments.

Reply:
Ok, thanks. I'll go with a NOTE and remove from the sample.